tests/func/test_init.py

```diff
@@ -111,3 +111,17 @@ def test_gen_dvcignore(tmp_dir):
         "# https://dvc.org/doc/user-guide/dvcignore\n"
     )
     assert text == (tmp_dir / ".dvcignore").read_text()
+
+
+def test_init_when_ignored_by_git(tmp_dir, scm, monkeypatch, caplog):
```

Comment:
I don't think the `monkeypatch` fixture is needed for this test

Reply:
Good catch. Removing it.